improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -808,6 +813,12 @@ def _location_and_scale_parameters_to_percentiles(
 
         percentile_cube = create_cube_with_percentiles(
             percentiles, template_slice, result)
+        # Make the mask defined above fit the data size and then apply to the
+        # probability cube.
+        stack_number = percentile_cube.data.shape[0]
+        mask_array = np.stack([mask]*stack_number)
+        percentile_cube.data = np.ma.masked_where(
+            mask_array == True, percentile_cube.data)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Comparison to True should be just 'expr'](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4922716)